lib/zone-spec/fake-async-test.ts

```diff
@@ -297,8 +297,15 @@
               break;
             case 'XMLHttpRequest.send':
               throw new Error('Cannot make XHRs from within a fake async test.');
+            case 'requestAnimationFrame':
+            case 'webkitRequestAnimationFrame':
+            case 'mozRequestAnimationFrame':
+              // Simulate a requestAnimationFrame by using a setTimeout with 16 ms.
+              // (60 frames per second)
+              task.data['handleId'] = this._setTimeout(task.invoke, 16, (task.data as any)['args']);
+              break;
             default:
-              task = delegate.scheduleTask(target, task);
+              throw new Error('Unknown macroTask scheduled in fake async test: ' + task.source);
```

Comment:
@vikerman , @juliemr, I have one question that if we throw Error, `unknown macroTask` will not be able to be run in `fakeAsyncZone`, currently beside `setTimeout/Interval` and `requestAnimationFrame`, other zone patched macroTasks include `setImmediate` and `nodejs fs/crypto` methods, should we also make them available in the `fakeAsyncTest`? 

thank you!